workers/data_refinery_workers/processors/no_op.yml

```diff
@@ -0,0 +1,34 @@
+---  # "Submitter-processed" processor
+
+# Operating system distribution (such as Ubuntu 16.04, CentOS 7.2, etc)
+os_distribution:   # DO NOT forget the trailing ':'
+
+# OS-level packages (installed by OS package manager)
+os_pkg:
+  - python3
+  - python3-pip
+  - r-base
+
+# Python pip packages
+python:
+  - Django
+  - data-refinery-common
+
+# User-installed R packages
+R:
+  - AnnotationDbi
+  - Bioconductor
+  - tidyverse
+  - rlang
+  - illuminaHumanv1.db
+  - illuminaHumanv2.db
+  - illuminaHumanv3.db
+  - illuminaHumanv4.db
+  - illuminaMousev1.db
+  - illuminaMousev1p1.db
+  - illuminaMousev2.db
+  - illuminaRatv1.db
+
+# Any other stuff
+other:
```

Comment:
For what it's worth, I can see why you did this. I wonder if it really makes sense to write code for each edge case instead of grabbing `stdout` by running `cat` (or allowing another arbitrary command line - this could be a superset of the things above where you run --version - could that grab arbitrary output and then you specify the version command then).

Reply:
In the case of `zenodo_url`, we just need to grab `cat <filename>` output. What if there are other information that we want to keep track of but it is not saved in a file? In those cases we may need to launch some other commands. I am using `cmd_line` to specify any user-installed programs that are launched by a command line, and `other` for anything else.

Does this make sense?